app/views/recipe/show.html.erb

```diff
@@ -0,0 +1,49 @@
+<h1 class="text-center margin-buttom"><%= @recipe.name%> <%= @recipe.id%> </h1>
+<div>
+  <div class="d-flex justify-content-around">
+  <div>
+   <p>preparation_time: <%= @recipe.preparation_time%>
+  <p>cooking_time: <%= @recipe.cooking_time%></p>
+  </div>
+    <div class="d-flex justify-content-start">
+      <div class="d-flex flex-column">
+        <h3 style="color: green; font-weight: 800"><%= @recipe.name%></h3>
+        <h4>Public: <%= @recipe.public_recipe%></h4>
+            <p>description: <%= @recipe.description%></p>
+      </div>
+    </div>
+  </div>
+  <div class="d-flex justify-content-start" style="margin-left: 15rem">
+<button type="button" class="btn btn-dark" data-bs-toggle="modal" data-bs-target="#staticBackdrop">
+    Generate shopping list
+</button>
+  </div>
+
+<!-- Modal -->
+<div class="modal fade" id="staticBackdrop" data-bs-backdrop="static" data-bs-keyboard="false" tabindex="-1" aria-labelledby="staticBackdropLabel" aria-hidden="true">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h1 class="modal-title fs-5" id="staticBackdropLabel">Generating Shopping List</h1>
+        <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+      </div>
+      <div class="modal-body">
+    <%= form_with  url: shopping_list_path, method: :post, local: true, scope: :@shopping do |f| %>
+    <div class="form-group">
+      <h3>not ina a requirement</h3>
+        <div class="submit-btn">
+             <%= f.submit  class: "btn btn-primary btn-lg btn-block"  %>
+        </div>
+    </div>
+    <% end %>
+      </div>
+      <div class="modal-footer">
+        <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">Close</button>
+        <button type="button" class="btn btn-primary">Understood</button>
+      </div>
+    </div>
+  </div>
+</div>
```

Comment:
@mirouhml hello bro:

yeah we just add the pop up button there and didnt add any functionality because it was in a requirement that this `generating Shopping List` is for only less than 3 members. as you can see also in a pic :arrow_double_down: 

![Screenshot from 2022-12-17 05-28-51](https://user-images.githubusercontent.com/104095543/208213129-d45790f5-9965-41c4-a905-df8d49d9cad9.png)